dev/breeze/src/airflow_breeze/utils/run_tests.py

```diff
@@ -394,6 +394,10 @@ def generate_args_for_pytest(
                 "--disable-warnings",
             ]
         )
+    else:
+        # Avoid edge cases when there are no available tests, e.g. No-Pydantic for Weaviate provider.
+        # https://docs.pytest.org/en/stable/reference/exit-codes.html
+        args.append("--suppress-no-test-exit-code")
```

Comment:
On second thought - maybe  better to enable it only for "special" tests @Taragolis ? This should never happen - I think for regular tests.

Reply:
Done